sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/resource/ResourceMessage.groovy

```diff
@@ -0,0 +1,28 @@
+package life.qbic.portal.sampletracking.resource
+
+import life.qbic.portal.sampletracking.communication.Message
+
+
+/**
+ * <p>A message that communicates changes in a resource of type T</p>
+ *
+ * @param <T>  the data type stored in the resource
+ * @since 1.0.0
+ */
+abstract class ResourceMessage<T> extends Message {
```

Comment:
I dont see the benefit of Message here. It would be sufficient to know its a ResourceMessage